config/src/test/kotlin/maru/config/HopliteFriendlinessTest.kt

```diff
@@ -166,6 +166,7 @@ class HopliteFriendlinessTest {
   private val qbftOptions =
     """
     communication-margin=100m
+    min-block-build-time=1000m
```

Comment:
Should we instead pull it from the protocol's block time?

Reply:
no, it should be a smaller so that there is sufficient time to start building the next block for the delayed block creator for the next block. I just chose a number for testing the config that was the same as the block time by coincidence.

Reply:
Then isn't it equivalent to blockTime - communicationMargin which we already had before your changes?